plugins/catalog-backend/src/database/DefaultProcessingDatabase.test.ts

```diff
@@ -1447,10 +1447,12 @@ describe('Default Processing Database', () => {
         const result1 = await db.transaction(async tx =>
           db.listParents(tx, { entityRef: 'component:default/foobar' }),
         );
-        expect(result1.entityRefs).toEqual([
-          'location:default/root-1',
-          'location:default/root-2',
-        ]);
+        expect(result1.entityRefs).toEqual(
+          expect.arrayContaining([
```

Comment:
Hmm this doesn't seem right. The new test is _weaker_ than the old one (it allows for more entries than the two expected ones), and that would probably lead to problems

Reply:
Yeah, it's a problem with Postgres9, which is treating the indices differently when it's a string vs text. Not sure why that is. I think the only way around this is to switch it to string when it's MySQL. Since textType for text defaults to TEXT, the max size should be the same. But def. correct me if I'm wrong.